uaclient/apt.py

```diff
@@ -340,6 +341,27 @@ def run_apt_update_command(
     return out
 
 
+def update_sources_list(sources_list: str):
+    retry_sleeps = APT_RETRIES.copy()
+    try:
+        while True:
```

Comment:
Could we use the `@util.retry` decorator to simplify the logic here?

Reply:
Thanks for pointing this out, refactored.